app/Resources/views/blog/index.html.twig

```diff
@@ -14,6 +14,11 @@
             {{ post.summary|md2html }}
         </article>
     {% endfor %}
+
+    {# display navigation #}
```

Comment:
This comment probably should be removed because the code below it's very easy to understand.